### ISSUE TITLE
Solr: can't use version 7.5.0 (only 7.2.1 works)

### ISSUE THREAD

**Issue by julie-sullivan** [intermine/intermine#1970]
```
 Task :dbmodel:postProcess
Performing postprocess create-search-index.
[corePostProcess] SLF4J: Class path contains multiple SLF4J bindings.
[corePostProcess] SLF4J: Found binding in [jar:file:/home/shokin/.gradle/caches/modules-2/files-2.1/org.slf4j/slf4j-simple/1.7.7/8095d0b9f7e0a9cd79a663c740e0f8fb31d0e2c8/slf4j-simple-1.7.7.jar!/org/slf4j/impl/StaticLoggerBinder.class]
[corePostProcess] SLF4J: Found binding in [jar:file:/home/shokin/.gradle/caches/modules-2/files-2.1/org.slf4j/slf4j-jdk14/1.7.10/13675a57601d9106fe82314fe9eebb9b2d9b13c6/slf4j-jdk14-1.7.10.jar!/org/slf4j/impl/StaticLoggerBinder.class]
[corePostProcess] SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
[corePostProcess] SLF4J: Actual binding is of type [org.slf4j.impl.SimpleLoggerFactory]
[corePostProcess] [Task worker for ':'] INFO com.zaxxer.hikari.HikariDataSource - db.production - Starting...
[corePostProcess] [Task worker for ':'] INFO com.zaxxer.hikari.HikariDataSource - db.production - Start completed.
[corePostProcess] Creating index for keyword search...
[corePostProcess] Creating keyword index failed
[corePostProcess] org.apache.solr.client.solrj.impl.HttpSolrClient$RemoteExecutionException: Error from server at http://localhost:8983/solr/beanmine-search: error processing commands
[corePostProcess]       at org.apache.solr.client.solrj.impl.HttpSolrClient$RemoteExecutionException.create(HttpSolrClient.java:829)
[corePostProcess]       at org.apache.solr.client.solrj.impl.HttpSolrClient.executeMethod(HttpSolrClient.java:620)
[corePostProcess]       at org.apache.solr.client.solrj.impl.HttpSolrClient.request(HttpSolrClient.java:255)
[corePostProcess]       at org.apache.solr.client.solrj.impl.HttpSolrClient.request(HttpSolrClient.java:244)
[corePostProcess]       at org.apache.solr.client.solrj.SolrRequest.process(SolrRequest.java:194)
[corePostProcess]       at org.apache.solr.client.solrj.SolrRequest.process(SolrRequest.java:211)
[corePostProcess]       at org.intermine.api.searchengine.solr.SolrIndexHandler.createFieldTypeDefinitions(SolrIndexHandler.java:304)
[corePostProcess]       at org.intermine.api.searchengine.solr.SolrIndexHandler.createIndex(SolrIndexHandler.java:65)
[corePostProcess]       at org.intermine.bio.postprocess.CreateSearchIndexProcess.postProcess(CreateSearchIndexProcess.java:66)
[corePostProcess]       at org.intermine.task.PostProcessorTask.execute(PostProcessorTask.java:70)
[corePostProcess]       at org.apache.tools.ant.UnknownElement.execute(UnknownElement.java:293)
[corePostProcess]       at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[corePostProcess]       at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[corePostProcess]       at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[corePostProcess]       at java.lang.reflect.Method.invoke(Method.java:498)
[corePostProcess]       at org.apache.tools.ant.dispatch.DispatchUtils.execute(DispatchUtils.java:106)
[corePostProcess]       at groovy.util.AntBuilder.performTask(AntBuilder.java:327)
[corePostProcess]       at groovy.util.AntBuilder.nodeCompleted(AntBuilder.java:272)
[corePostProcess]       at groovy.util.BuilderSupport.doInvokeMethod(BuilderSupport.java:154)
[corePostProcess]       at groovy.util.AntBuilder.doInvokeMethod(AntBuilder.java:207)
[corePostProcess]       at groovy.util.BuilderSupport.invokeMethod(BuilderSupport.java:67)
[corePostProcess]       at org.codehaus.groovy.runtime.callsite.PogoMetaClassSite.call(PogoMetaClassSite.java:48)
[corePostProcess]       at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:48)
[corePostProcess]       at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:113)
[corePostProcess]       at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:125)
[corePostProcess]       at org.intermine.plugin.postprocess.PostProcessPlugin$_apply_closure2$_closure6$_closure7.doCall(PostProcessPlugin.groovy:127)
[corePostProcess]       at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[corePostProcess]       at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[corePostProcess]       at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[corePostProcess]       at java.lang.reflect.Method.invoke(Method.java:498)
[corePostProcess]       at org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:93)
[corePostProcess]       at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:325)
[corePostProcess]       at org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:294)
[corePostProcess]       at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1022)
[corePostProcess]       at groovy.lang.Closure.call(Closure.java:414)
[corePostProcess]       at groovy.lang.Closure.call(Closure.java:430)
[corePostProcess]       at org.codehaus.groovy.runtime.DefaultGroovyMethods.each(DefaultGroovyMethods.java:2040)
[corePostProcess]       at org.codehaus.groovy.runtime.DefaultGroovyMethods.each(DefaultGroovyMethods.java:2025)
[corePostProcess]       at org.codehaus.groovy.runtime.DefaultGroovyMethods.each(DefaultGroovyMethods.java:2066)
[corePostProcess]       at org.codehaus.groovy.runtime.dgm$162.invoke(Unknown Source)
[corePostProcess]       at org.codehaus.groovy.runtime.callsite.PojoMetaMethodSite$PojoMetaMethodSiteNoUnwrapNoCoerce.invoke(PojoMetaMethodSite.java:274)
[corePostProcess]       at org.codehaus.groovy.runtime.callsite.PojoMetaMethodSite.call(PojoMetaMethodSite.java:56)
[corePostProcess]       at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:48)
[corePostProcess]       at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:113)
[corePostProcess]       at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:125)
[corePostProcess]       at org.intermine.plugin.postprocess.PostProcessPlugin$_apply_closure2$_closure6.doCall(PostProcessPlugin.groovy:67)
[corePostProcess]       at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[corePostProcess]       at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[corePostProcess]       at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[corePostProcess]       at java.lang.reflect.Method.invoke(Method.java:498)
[corePostProcess]       at org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:93)
[corePostProcess]       at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:325)
[corePostProcess]       at org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:294)
[corePostProcess]       at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1022)
[corePostProcess]       at groovy.lang.Closure.call(Closure.java:414)
[corePostProcess]       at groovy.lang.Closure.call(Closure.java:430)
[corePostProcess]       at org.gradle.api.internal.AbstractTask$ClosureTaskAction.execute(AbstractTask.java:681)
[corePostProcess]       at org.gradle.api.internal.AbstractTask$ClosureTaskAction.execute(AbstractTask.java:656)
[corePostProcess]       at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter$1.run(ExecuteActionsTaskExecuter.java:122)
[corePostProcess]       at org.gradle.internal.progress.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:336)
[corePostProcess]       at org.gradle.internal.progress.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:328)
[corePostProcess]       at org.gradle.internal.progress.DefaultBuildOperationExecutor.execute(DefaultBuildOperationExecutor.java:197)
[corePostProcess]       at org.gradle.internal.progress.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:107)
[corePostProcess]       at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeAction(ExecuteActionsTaskExecuter.java:111)
[corePostProcess]       at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeActions(ExecuteActionsTaskExecuter.java:92)
[corePostProcess]       at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.execute(ExecuteActionsTaskExecuter.java:70)
[corePostProcess]       at org.gradle.api.internal.tasks.execution.SkipUpToDateTaskExecuter.execute(SkipUpToDateTaskExecuter.java:63)
[corePostProcess]       at org.gradle.api.internal.tasks.execution.ResolveTaskOutputCachingStateExecuter.execute(ResolveTaskOutputCachingStateExecuter.java:54)
[corePostProcess]       at org.gradle.api.internal.tasks.execution.ValidatingTaskExecuter.execute(ValidatingTaskExecuter.java:58)
[corePostProcess]       at org.gradle.api.internal.tasks.execution.SkipEmptySourceFilesTaskExecuter.execute(SkipEmptySourceFilesTaskExecuter.java:88)
[corePostProcess]       at org.gradle.api.internal.tasks.execution.ResolveTaskArtifactStateTaskExecuter.execute(ResolveTaskArtifactStateTaskExecuter.java:52)
[corePostProcess]       at org.gradle.api.internal.tasks.execution.SkipTaskWithNoActionsExecuter.execute(SkipTaskWithNoActionsExecuter.java:52)
[corePostProcess]       at org.gradle.api.internal.tasks.execution.SkipOnlyIfTaskExecuter.execute(SkipOnlyIfTaskExecuter.java:54)
[corePostProcess]       at org.gradle.api.internal.tasks.execution.ExecuteAtMostOnceTaskExecuter.execute(ExecuteAtMostOnceTaskExecuter.java:43)
[corePostProcess]       at org.gradle.api.internal.tasks.execution.CatchExceptionTaskExecuter.execute(CatchExceptionTaskExecuter.java:34)
[corePostProcess]       at org.gradle.execution.taskgraph.DefaultTaskGraphExecuter$EventFiringTaskWorker$1.run(DefaultTaskGraphExecuter.java:248)
[corePostProcess]       at org.gradle.internal.progress.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:336)
[corePostProcess]       at org.gradle.internal.progress.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:328)
[corePostProcess]       at org.gradle.internal.progress.DefaultBuildOperationExecutor.execute(DefaultBuildOperationExecutor.java:197)
[corePostProcess]       at org.gradle.internal.progress.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:107)
[corePostProcess]       at org.gradle.execution.taskgraph.DefaultTaskGraphExecuter$EventFiringTaskWorker.execute(DefaultTaskGraphExecuter.java:241)
[corePostProcess]       at org.gradle.execution.taskgraph.DefaultTaskGraphExecuter$EventFiringTaskWorker.execute(DefaultTaskGraphExecuter.java:230)
[corePostProcess]       at org.gradle.execution.taskgraph.DefaultTaskPlanExecutor$TaskExecutorWorker.processTask(DefaultTaskPlanExecutor.java:124)
[corePostProcess]       at org.gradle.execution.taskgraph.DefaultTaskPlanExecutor$TaskExecutorWorker.access$200(DefaultTaskPlanExecutor.java:80)
[corePostProcess]       at org.gradle.execution.taskgraph.DefaultTaskPlanExecutor$TaskExecutorWorker$1.execute(DefaultTaskPlanExecutor.java:105)
[corePostProcess]       at org.gradle.execution.taskgraph.DefaultTaskPlanExecutor$TaskExecutorWorker$1.execute(DefaultTaskPlanExecutor.java:99)
[corePostProcess]       at org.gradle.execution.taskgraph.DefaultTaskExecutionPlan.execute(DefaultTaskExecutionPlan.java:625)
[corePostProcess]       at org.gradle.execution.taskgraph.DefaultTaskExecutionPlan.executeWithTask(DefaultTaskExecutionPlan.java:580)
[corePostProcess]       at org.gradle.execution.taskgraph.DefaultTaskPlanExecutor$TaskExecutorWorker.run(DefaultTaskPlanExecutor.java:99)
[corePostProcess]       at org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:63)
[corePostProcess]       at org.gradle.internal.concurrent.ManagedExecutorImpl$1.run(ManagedExecutorImpl.java:46)
[corePostProcess]       at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
[corePostProcess]       at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
[corePostProcess]       at org.gradle.internal.concurrent.ThreadFactoryImpl$ManagedThreadRunnable.run(ThreadFactoryImpl.java:55)
[corePostProcess]       at java.lang.Thread.run(Thread.java:748)


BUILD SUCCESSFUL in 7s
8 actionable tasks: 5 executed, 3 up-to-date
[Thread-41] INFO com.zaxxer.hikari.HikariDataSource - db.production - Shutdown initiated...
[Thread-41] INFO com.zaxxer.hikari.HikariDataSource - db.production - Shutdown completed.


AND

Performing postprocess create-autocomplete-index.
[corePostProcess] SLF4J: Class path contains multiple SLF4J bindings.
[corePostProcess] SLF4J: Found binding in [jar:file:/home/shokin/.gradle/caches/modules-2/files-2.1/org.slf4j/slf4j-simple/1.7.7/8095d0b9f7e0a9cd79a663c740e0f8fb31d0e2c8/slf4j-simple-1.7.7.jar!/org/slf4j/impl/StaticLoggerBinder.class]
[corePostProcess] SLF4J: Found binding in [jar:file:/home/shokin/.gradle/caches/modules-2/files-2.1/org.slf4j/slf4j-jdk14/1.7.10/13675a57601d9106fe82314fe9eebb9b2d9b13c6/slf4j-jdk14-1.7.10.jar!/org/slf4j/impl/StaticLoggerBinder.class]
[corePostProcess] SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
[corePostProcess] SLF4J: Actual binding is of type [org.slf4j.impl.SimpleLoggerFactory]
[corePostProcess] [Task worker for ':'] INFO com.zaxxer.hikari.HikariDataSource - db.production - Starting...
[corePostProcess] [Task worker for ':'] INFO com.zaxxer.hikari.HikariDataSource - db.production - Start completed.
[corePostProcess] create autocomplete index ...
[corePostProcess] Indexing SOTerm.name
[corePostProcess] Indexing OntologyTerm.name


FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':dbmodel:postProcess'.
> org.apache.solr.client.solrj.impl.HttpSolrClient$RemoteExecutionException: Error from server at http://localhost:8983/solr/beanmine-autocomplete: error processing commands

* Try:
Run with --info or --debug option to get more log output.

* Exception is:
org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':dbmodel:postProcess'.
        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeActions(ExecuteActionsTaskExecuter.java:100)
        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.execute(ExecuteActionsTaskExecuter.java:70)
        at org.gradle.api.internal.tasks.execution.SkipUpToDateTaskExecuter.execute(SkipUpToDateTaskExecuter.java:63)
        at org.gradle.api.internal.tasks.execution.ResolveTaskOutputCachingStateExecuter.execute(ResolveTaskOutputCachingStateExecuter.java:54)
        at org.gradle.api.internal.tasks.execution.ValidatingTaskExecuter.execute(ValidatingTaskExecuter.java:58)
        at org.gradle.api.internal.tasks.execution.SkipEmptySourceFilesTaskExecuter.execute(SkipEmptySourceFilesTaskExecuter.java:88)
        at org.gradle.api.internal.tasks.execution.ResolveTaskArtifactStateTaskExecuter.execute(ResolveTaskArtifactStateTaskExecuter.java:52)
        at org.gradle.api.internal.tasks.execution.SkipTaskWithNoActionsExecuter.execute(SkipTaskWithNoActionsExecuter.java:52)
        at org.gradle.api.internal.tasks.execution.SkipOnlyIfTaskExecuter.execute(SkipOnlyIfTaskExecuter.java:54)
        at org.gradle.api.internal.tasks.execution.ExecuteAtMostOnceTaskExecuter.execute(ExecuteAtMostOnceTaskExecuter.java:43)
        at org.gradle.api.internal.tasks.execution.CatchExceptionTaskExecuter.execute(CatchExceptionTaskExecuter.java:34)
        at org.gradle.execution.taskgraph.DefaultTaskGraphExecuter$EventFiringTaskWorker$1.run(DefaultTaskGraphExecuter.java:248)
        at org.gradle.internal.progress.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:336)
        at org.gradle.internal.progress.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:328)
        at org.gradle.internal.progress.DefaultBuildOperationExecutor.execute(DefaultBuildOperationExecutor.java:197)
        at org.gradle.internal.progress.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:107)
        at org.gradle.execution.taskgraph.DefaultTaskGraphExecuter$EventFiringTaskWorker.execute(DefaultTaskGraphExecuter.java:241)
        at org.gradle.execution.taskgraph.DefaultTaskGraphExecuter$EventFiringTaskWorker.execute(DefaultTaskGraphExecuter.java:230)
        at org.gradle.execution.taskgraph.DefaultTaskPlanExecutor$TaskExecutorWorker.processTask(DefaultTaskPlanExecutor.java:124)
        at org.gradle.execution.taskgraph.DefaultTaskPlanExecutor$TaskExecutorWorker.access$200(DefaultTaskPlanExecutor.java:80)
        at org.gradle.execution.taskgraph.DefaultTaskPlanExecutor$TaskExecutorWorker$1.execute(DefaultTaskPlanExecutor.java:105)
        at org.gradle.execution.taskgraph.DefaultTaskPlanExecutor$TaskExecutorWorker$1.execute(DefaultTaskPlanExecutor.java:99)
        at org.gradle.execution.taskgraph.DefaultTaskExecutionPlan.execute(DefaultTaskExecutionPlan.java:625)
        at org.gradle.execution.taskgraph.DefaultTaskExecutionPlan.executeWithTask(DefaultTaskExecutionPlan.java:580)
        at org.gradle.execution.taskgraph.DefaultTaskPlanExecutor$TaskExecutorWorker.run(DefaultTaskPlanExecutor.java:99)
        at org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:63)
        at org.gradle.internal.concurrent.ManagedExecutorImpl$1.run(ManagedExecutorImpl.java:46)
        at org.gradle.internal.concurrent.ThreadFactoryImpl$ManagedThreadRunnable.run(ThreadFactoryImpl.java:55)
Caused by: : org.apache.solr.client.solrj.impl.HttpSolrClient$RemoteExecutionException: Error from server at http://localhost:8983/solr/beanmine-autocomplete: error processing commands
        at org.intermine.task.PostProcessorTask.execute(PostProcessorTask.java:73)
        at org.apache.tools.ant.UnknownElement.execute(UnknownElement.java:293)
        at org.apache.tools.ant.dispatch.DispatchUtils.execute(DispatchUtils.java:106)
        at org.intermine.plugin.postprocess.PostProcessPlugin$_apply_closure2$_closure6$_closure7.doCall(PostProcessPlugin.groovy:127)
        at org.intermine.plugin.postprocess.PostProcessPlugin$_apply_closure2$_closure6.doCall(PostProcessPlugin.groovy:67)
        at org.gradle.api.internal.AbstractTask$ClosureTaskAction.execute(AbstractTask.java:681)
        at org.gradle.api.internal.AbstractTask$ClosureTaskAction.execute(AbstractTask.java:656)
        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter$1.run(ExecuteActionsTaskExecuter.java:122)
        at org.gradle.internal.progress.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:336)
        at org.gradle.internal.progress.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:328)
        at org.gradle.internal.progress.DefaultBuildOperationExecutor.execute(DefaultBuildOperationExecutor.java:197)
        at org.gradle.internal.progress.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:107)
        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeAction(ExecuteActionsTaskExecuter.java:111)
        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeActions(ExecuteActionsTaskExecuter.java:92)
        ... 27 more
Caused by: org.apache.solr.client.solrj.impl.HttpSolrClient$RemoteExecutionException: Error from server at http://localhost:8983/solr/beanmine-autocomplete: error processing commands
        at org.apache.solr.client.solrj.impl.HttpSolrClient$RemoteExecutionException.create(HttpSolrClient.java:829)
        at org.apache.solr.client.solrj.impl.HttpSolrClient.executeMethod(HttpSolrClient.java:620)
        at org.apache.solr.client.solrj.impl.HttpSolrClient.request(HttpSolrClient.java:255)
        at org.apache.solr.client.solrj.impl.HttpSolrClient.request(HttpSolrClient.java:244)
        at org.apache.solr.client.solrj.SolrRequest.process(SolrRequest.java:194)
        at org.apache.solr.client.solrj.SolrRequest.process(SolrRequest.java:211)
        at org.intermine.web.autocompletion.AutoCompleter.buildIndex(AutoCompleter.java:244)
        at org.intermine.bio.postprocess.CreateAutocompleteIndexProcess.postProcess(CreateAutocompleteIndexProcess.java:53)
        at org.intermine.task.PostProcessorTask.execute(PostProcessorTask.java:70)
        ... 40 more


* Get more help at https://help.gradle.org

BUILD FAILED in 9s
8 actionable tasks: 5 executed, 3 up-to-date
[Thread-41] INFO com.zaxxer.hikari.HikariDataSource - db.production - Shutdown initiated...
[Thread-41] INFO com.zaxxer.hikari.HikariDataSource - db.production - Shutdown completed. 
```

**Comment by arunans23** [comment]
@julie-sullivan 

I tried the biotestmine with Solr 7.6.0 and I encountered the same problem. I went over the Solr logs (which is more detailed) and found out that, an exception is thrown when same field name or field type got created again and again. For example, every time we run the postprocess, the fieldname "Category" is trying to get created. 

In Solr 7.2.1, it was not the case. If a field name already existed, it just ignored it (or over-rid it).

**Comment by julie-sullivan** [comment]
Fantastic sleuthing @arunans23! So if we, in our search index code, check for duplicates, that should fix the problem?

I only see this:

```
            IndexHandler indexHandler = new SolrIndexHandler();
            indexHandler.createIndex(osw, classKeys);
```

Doesn't seem like we have a lot of control over duplicates? Oh, are the duplicates in the class keys?

**Comment by arunans23** [comment]
Actually the problem is not only because of the duplicates. Even when you run the index process twice, this problem will occur because we are not deleting the field names from the managed schema for every index.

We might have to change the logic [here](https://github.com/intermine/intermine/blob/dev/intermine/api/src/main/java/org/intermine/api/searchengine/solr/SolrIndexHandler.java#L203), if we want to make it work with new solr versions. 

In the Schema API [page](https://lucene.apache.org/solr/guide/7_6/schema-api.html#delete-a-field) of Solr, there are two methods : replace and delete field name. But both of them returns an exception when the field name doesn't exist. Hence we have a problem here. We need to come up with a solution for both of the scenarios below.

- When a fresh collection is created, fieldnames and fieldtypes would not already exist. Hence they have to be created.
- When the index process is run again and again, fieldnames might already be defined in the schema. In that case creating part has to be ignored.

**Comment by arunans23** [comment]
@julie-sullivan I think I have given a solution to the issue. I have modified the code to check the existing schema in Solr, before giving any updates to the Solr. I have checked the solution with Solr 7.2.1 and Solr 7.6.0. Please review the changes and merge them if you are satisfied. 

Basically this issue came up because Solr fixed one of their bugs in 7.3. https://issues.apache.org/jira/browse/SOLR-11631

Btw, I wanted to do this ASAP, but got stuck with some tomcat issues. Thank you @danielabutano for helping me to deploy the biotestmine war in tomcat server.
